kvs-conditional-read/README.md

```diff
@@ -0,0 +1,5 @@
+## Rewrite request URI
+
+**CloudFront Functions event type: viewer request**
+
+This example provides a dynamic request URI rewriting mechanism, allowing for A/B testing or gradual rollout of application versions, while also maintaining user stickiness to ensure a consistent experience. It rewrites the request URI based on a configuration stored in CloudFront KeyValueStore. Using your KeyValueStore ID in the code is optional.
```

Comment:
I think we should remove the last sentence about storing the KVS ID in the code. I think we should instead just say that you have to create a KVS and put the secret in there and then attach the KVS to the function. I don't quite know the level of detail that we need to go into how to setup KVS for this readme, but if the KVS ID is optional then we should not even mention it since its much easier without it in the code.